apps/server/src/modules/collaborative-text-editor/service/collaborative-text-editor.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { Logger } from '@src/core/logger';
+import { EtherpadClientAdapter } from '@src/infra/etherpad-client';
+import { ServerConfig } from '@src/modules/server';
+import { GetCollaborativeTextEditorForParentParams } from '../api/dto/get-collaborative-text-editor-for-parent.params';
+import { CollaborativeTextEditor } from '../domain/do/collaborative-text-editor';
+
+@Injectable()
+export class CollaborativeTextEditorService {
+	constructor(
+		private readonly collaborativeTextEditorAdapter: EtherpadClientAdapter,
+		private readonly configService: ConfigService<ServerConfig, true>,
+		private logger: Logger
+	) {
+		this.logger.setContext(CollaborativeTextEditorService.name);
+	}
+
+	async createCollaborativeTextEditor(
```

Comment:
This method is called "create..." while those in UC and Controller are called "get...". I think calling them "getOrCreate..." everywhere would be best.